pkg/driver/controller.go

```diff
@@ -130,6 +130,10 @@ func (d *ControllerService) CreateVolume(ctx context.Context, req *csi.CreateVol
 
 	for key, value := range req.GetParameters() {
 		switch strings.ToLower(key) {
+		// we tell user here that they should use csi.storage.k8s.io/fstype instead but there is no switch cases to handle that
+		// Suggestion to add case FSTypeKey that will set the fstype and in the case "fstype" we print the warning and still set
+		// fstype for them
+
```

Comment:
Start of discussion thread

Reply:
`csi.storage.k8s.io/fstype` is a special parameter that causes the `external-provisioner` to do something special, so not handling it here is intentional: https://kubernetes-csi.github.io/docs/external-provisioner.html#storageclass-parameters

We've had this `fstype` warning for so long though that it probably should be removed at some point but I don't think this PR is the appropriate place.